engine/src/main/java/org/hibernate/search/engine/spi/DocumentBuilderIndexedEntity.java

```diff
@@ -853,18 +858,12 @@ private void buildDocumentFieldForClassBridges(Document doc,
 			FieldBridge fieldBridge = fieldMetadata.getFieldBridge();
 			final String fieldName = fieldMetadata.getAbsoluteName();
 			final FieldBridge oneWayConversionContext = conversionContext.oneWayConversionContext( fieldBridge );
-			conversionContext.pushProperty( fieldName );
-			try {
-				oneWayConversionContext.set(
-						fieldName,
-						unproxiedInstance,
-						doc,
-						typeMetadata.getClassLuceneOptions( fieldMetadata, documentBoost )
-				);
-			}
-			finally {
-				conversionContext.popProperty();
-			}
+			oneWayConversionContext.set(
```

Comment:
Here is the commit: https://github.com/hibernate/hibernate-search/pull/1217/commits/966d90a45940f9edddbd8149263fb742971a83ac
(`HSEARCH-2457 Make sure ContextualExceptionBridgeHelper provides as much information as possible in exception messages`)